src/sinks/websocket/sink.rs

```diff
@@ -236,12 +236,12 @@ impl WebSocketSink {
 
     const fn should_encode_as_binary(&self) -> bool {
         use vector_lib::codecs::encoding::Serializer::{
-            Avro, Csv, Gelf, Json, Logfmt, Native, NativeJson, Protobuf, RawMessage, Text,
```

Comment:
Is this changes also related to the `cargo check`?

Reply:
Yes